functions/zoom-downloader.py

```diff
@@ -36,7 +37,7 @@ class PermanentDownloadError(Exception):
     pass
 
 
-class ZoomFileAccessError(Exception):
+class ZoomDownloadLinkError(Exception):
```

Comment:
Changed exception name to be more general since we're giving up on telling the difference between types of download link failures (access vs deleted file)